docs/docs/render-props.md

```diff
@@ -0,0 +1,226 @@
+---
+id: render-props
+title: Render Props
+permalink: docs/render-props.html
+---
+
+The term ["render prop"](https://cdb.reacttraining.com/use-a-render-prop-50de598f11ce) refers to an advanced technique for sharing code between React components using a prop whose value is a function instead of a resorting to using a [higher-order component](/react/docs/higher-order-components.html) (HOC).
+
+## Use Render Props for Cross-Cutting Concerns
+
+A render prop is a simple, elegant way to resolve cross-cutting concerns in your React app without [the caveats of higher-order components](/react/docs/higher-order-components.html#caveats).
+
+Components are the primary unit of code reuse in React, but it's not always obvious how to share the state or behavior that one component encapsulates to other components that need that same state.
+
+For example, the following component tracks the mouse cursor position in a web app:
```

Comment:
"tracks the cursor position" would be fine.  I think it's fair to assume someone who speaks JS understands how `mousemove` works.